packages/gamut/src/Form/Input.tsx

```diff
@@ -56,8 +57,14 @@ export interface InputWrapperProps extends InputProps {
 
 /**  We greatly prefer NOT to do this but ReactRecurly has some specific needs around focus-styles + padding that force us to export them seperately. If we ever stop using React-Recurly, this code will be 🔪.
  *tldr: Do not do this unless you have already talked to Web-Plat and have failed to find any alternate (and better) solutions. */
-export const reactRecurlyFormFieldFocusStyles = formFieldFocusStyles;
-export const reactRecurlyFormFieldPaddingStyles = formFieldPaddingStyles;
+
+export const reactRecurlyFormFieldFocusStyles = system.css({
+  ...formFieldFocusStyles,
+});
```

Comment:
No issues with immutability here :) 

```suggestion
export const reactRecurlyFormFieldFocusStyles = system.css(formFieldFocusStyles);
```